healthmap/mriids2
# Update position of date range slider under charts

Fixes: https://github.com/healthmap/mriids2/issues/151

Test Case - 
- Pull down the code.
- Run locally or use http://PR-161-mriids.s3-website-us-east-1.amazonaws.com/
- Make sure the slider looks ok when switching between outbreaks as well as changing timespan
